Documentation/conf.py

```diff
@@ -87,7 +87,7 @@
 if not branch or branch == 'latest':
     branch = 'HEAD'
     archive_name = 'main'
-    chart_release = './cilium'
+    chart_release = './install/kubernetes/cilium'
```

Comment:
Can you please split this out, please? https://github.com/cilium/cilium/pull/26934 is being backported to all supported branches, so this change should likely be as well?

I'd also prefer having separate commits for the different issues that your PR addresses. Not a blocker because the commit remains small, but that would makes it easier to track the changes.

Reply:
(The updates themselves look good, by the way.)

Reply:
updated 🥰